app/helpers/scheduled_pickups_helper.rb

```diff
@@ -1,14 +1,5 @@
 module ScheduledPickupsHelper
   def format_pickup_time(scheduled_pickup)
-    start_at = scheduled_pickup.start_at
-    end_at = scheduled_pickup.end_at
-    time_format = "%l:%M %P"
-
-    t(
-      "scheduled_pickups.format",
-      weekday: Weekday.find(start_at.wday).label,
-      start_at: start_at.strftime(time_format),
-      end_at: end_at.strftime(time_format),
-    )
+    TimeRange.new(scheduled_pickup.start_at, scheduled_pickup.end_at)
```

Comment:
@gabebw this helper (and this newly extracted class) is indirectly tested at the [feature spec level](https://github.com/thoughtbot/freshfoodconnect/blob/1c3b400d3b46ae77aa1db469945fd133243653f2/spec/features/admin_views_pickup_times_spec.rb#L16).

This extraction poses a great opportunity for testing at a more fine-grained level.